src/teams/static.json

```diff
@@ -0,0 +1,123 @@
+[
+	{
+		"repos": [
+			"https://github.com/EclipseFdn/eclipsefdn-github-sync",
+			"https://github.com/EclipseFdn/events.eclipse.org",
+			"https://github.com/EclipseFdn/iot.eclipse.org",
+			"https://github.com/EclipseFdn/openvsx-publisher-agreement-specs",
+			"https://github.com/EclipseFdn/lts.eclipse.org",
+			"https://github.com/EclipseFdn/solstice-assets",
+			"https://github.com/EclipseFdn/openadx.eclipse.org",
+			"https://github.com/EclipseFdn/hugo-solstice-theme",
+			"https://github.com/EclipseFdn/EFDP",
+			"https://github.com/EclipseFdn/eclipsefdn-foundationdb-api-specs",
+			"https://github.com/EclipseFdn/eclipse.org-research",
+			"https://github.com/EclipseFdn/eclipsefdn-project-adopters",
+			"https://github.com/EclipseFdn/status.eclipse.org",
+			"https://github.com/EclipseFdn/react-eclipsefdn-members",
+			"https://github.com/EclipseFdn/projects-bots-api",
+			"https://github.com/EclipseFdn/sparkplug.eclipse.org",
+			"https://github.com/EclipseFdn/tangle.ee",
+			"https://github.com/EclipseFdn/ecdtools.eclipse.org",
+			"https://github.com/EclipseFdn/eclipsecon.org-archive",
+			"https://github.com/EclipseFdn/openmobility.eclipse.org",
+			"https://github.com/EclipseFdn/dockerfiles",
+			"https://github.com/EclipseFdn/openpass.eclipse.org",
+			"https://github.com/EclipseFdn/edgenative.eclipse.org",
+			"https://github.com/EclipseFdn/science.eclipse.org",
+			"https://github.com/EclipseFdn/gerrit-eca-plugin",
+			"https://github.com/EclipseFdn/git-eca-rest-api",
+			"https://github.com/EclipseFdn/jekyll-theme-eclipsefdn",
+			"https://github.com/EclipseFdn/geoip-rest-api",
+			"https://github.com/EclipseFdn/marketplace-rest-api",
+			"https://github.com/EclipseFdn/marketplace-rest-api-specs",
+			"https://github.com/EclipseFdn/planeteclipse.org",
+			"https://github.com/EclipseFdn/jquery-eclipsefdn-api",
+			"https://github.com/EclipseFdn/opengenesis.eclipse.org",
+			"https://github.com/EclipseFdn/EF-security-policy",
+			"https://github.com/EclipseFdn/cerberus",
+			"https://github.com/EclipseFdn/api.eclipse.org-docs",
+			"https://github.com/EclipseFdn/jenkins-pipeline-shared",
+			"https://github.com/EclipseFdn/eclipse-webhook",
+			"https://github.com/EclipseFdn/eclipsecon-data-scraper",
+			"https://github.com/EclipseFdn/EFSP",
+			"https://github.com/EclipseFdn/fedgeoday",
+			"https://github.com/EclipseFdn/locationtech-events"
+		],
+		"teamName": "eclipsefdn-staff",
+		"members": [
```

Comment:
We are missing a lot of people here. What am I missing?

Reply:
@autumnfound I will send you an email but I would expect to see 23 members on this list.